src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala

```diff
@@ -6,7 +6,7 @@ import akka.NotUsed
 import akka.stream.scaladsl.Source
 import com.google.inject.Inject
 import org.zalando.spearheads.innkeeper.api.JsonProtocols._
-import org.zalando.spearheads.innkeeper.api.{EskipRoute, EskipRouteWrapper, NewRoute, RouteName, RouteOut}
```

Comment:
You should use explicit imports whenever possible. You should instruct your IDE to use the explicit version.